### DISCUSSION TITLE
分享个 zig 开发的获取文件摘要信息的工具

### DISCUSSION THREAD

**Discussion by deatil** [zigcc/forum#59] [作品分享]
zig 现在做 web 还不是时候，又想用下 zig ，所以退而求其次用 zig 写点工具来熟悉下。

项目地址: https://github.com/deatil/ztool

当前工具主要是做文件的 md5 , sha1 这些的检测。也有对文件的编码解码，对文件信息做 md5, sha1 等的摘要获取

**Comment by jiacai2050** [comment]
为啥和要 web 过不去呢，zig 开发 web 没有任何优势吧？

**Reply by deatil** [reply]
zig打包的二进制更小。主要做得多的是web开发
